sdk/core/azure-core-tracing-opentelemetry/setup.py

```diff
@@ -55,6 +55,7 @@
     zip_safe=False,
     packages=[
         "azure.core.tracing.ext.opentelemetry_span",
+        "azure.core.tracing.ai.inference",
```

Comment:
what does it do?

Reply:
The packages argument in the setup() function in setup.py is used to specify all the Python import packages that should be included in the distribution package.

In other words, it lists all the Python modules that should be installed and made available when someone installs your package.

Reply:
thanks for the explanation! 

does it mean that anyone who installs `azure-core-tracing-opentelemetry` will need `azure-ai-inference`? This does not seem right.

Reply:
it is actually the tracing itself (notice the path has tracing). if this line is not here, then the tracing functionality itself won't be included in the resulting wheel.